### PR TITLE
RHEL-07-040460 - UsePrivilegeSeparation sandbox

### PR DESCRIPTION
**Pull request by konstruktoid**
Issue #2158

Signed-off-by: Thomas Sjögren <konstruktoid@users.noreply.github.com>

### CODE REVIEW

**Comment by shawndwells** [comment]
This reflects a bug in DISA's content. They were informed many, many times the correct setting is ``yes``, and DISA committed to fixing their content. It's unclear if they will hold to their commitment though.

We could update the XCCDF and OVAL to accept ``sandbox``, but the remediations should still set things to yes.

**Comment by shawndwells** [comment]
Derp. I was reading way to fast. The correct setting is ``sandbox``

File: RHEL/7/templates/static/oval/sshd_use_priv_separation.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@ root privileges when not needed.</description>
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sshd_use_priv_separation" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*(?i)UsePrivilegeSeparation(?-i)[\s]+yes[\s]*(?:|(?:#.*))?$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*(?i)UsePrivilegeSeparation(?-i)[\s]+(yes|sandbox)[\s]*(?:|(?:#.*))?$</ind:pattern>
```
Comment (shawndwells) [inline RHEL/7/templates/static/oval/sshd_use_priv_separation.xml]: Should we still accept yes?

(I'm learning towards no)

Comment (mpreisler) [inline RHEL/7/templates/static/oval/sshd_use_priv_separation.xml]: I agree, IMO we should only accept sandbox.

Comment (redhatrises) [inline RHEL/7/templates/static/oval/sshd_use_priv_separation.xml]: Probably should have both as (might be wrong on this) but I *think* that RHEL6 does not have sandbox mode. I'd say that this PR gets updated to use a variable that is then configured in the profiles.

Comment (mpreisler) [inline RHEL/7/templates/static/oval/sshd_use_priv_separation.xml]: @redhatrises this check is RHEL7 only

Comment (redhatrises) [inline RHEL/7/templates/static/oval/sshd_use_priv_separation.xml]: @mpreisler shoot me now..... this is not my week.

**Comment by jan-cerny** [comment]
Thanks.

**Pull request merged**